Documentation/config/core.txt

```diff
@@ -66,7 +66,11 @@ core.fsmonitor::
 	will identify all files that may have changed since the
 	requested date/time. This information is used to speed up git by
 	avoiding unnecessary processing of files that have not changed.
-	See the "fsmonitor-watchman" section of linkgit:githooks[5].
++
+See the "fsmonitor-watchman" section of linkgit:githooks[5].
```

Comment:
?? ... and see `core.fsmonitorHookVersion` ??

Reply:
Hmm. I don't think we need this here, but you're right, I need to mention the `core.useBuiltinFSMonitor` option in `githooks.txt`.